File: CS/WindowsUIForm.cs
using DevExpress.XtraEditors;
using System;
using System.Linq;
using System.Windows.Forms;

namespace dxExample
{
    public partial class WindowsUIForm : XtraForm {
        public WindowsUIForm() {
            InitializeComponent(); 
        }
        private void Form1_Shown(object sender, EventArgs e) {
            new FrameHoverHelper(tileContainer1.Info as ITileControl);
        }
        private void WindowsUIView1_QueryControl(object sender, DevExpress.XtraBars.Docking2010.Views.QueryControlEventArgs e) {
            e.Control = new Control();
        }
        private void tileContainer1_ButtonClick(object sender, DevExpress.XtraBars.Docking2010.ButtonEventArgs e) {
            TileControlForm tcForm = new TileControlForm();
            tcForm.Show();
        }
    }
}

File: VB/WindowsUIForm.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraEditors
Imports System
Imports System.Linq
Imports System.Windows.Forms

Namespace dxExample
	Partial Public Class WindowsUIForm
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub Form1_Shown(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Shown
			Dim TempFrameHoverHelper As FrameHoverHelper = New FrameHoverHelper(TryCast(tileContainer1.Info, ITileControl))
		End Sub
		Private Sub WindowsUIView1_QueryControl(ByVal sender As Object, ByVal e As DevExpress.XtraBars.Docking2010.Views.QueryControlEventArgs)
			e.Control = New Control()
		End Sub
		Private Sub tileContainer1_ButtonClick(ByVal sender As Object, ByVal e As DevExpress.XtraBars.Docking2010.ButtonEventArgs) Handles tileContainer1.ButtonClick
			Dim tcForm As New TileControlForm()
			tcForm.Show()
		End Sub
	End Class
End Namespace
